book/security.md

```diff
@@ -433,23 +431,18 @@ Cross-site requests
 ===================
 
 Cookies are site-specific, so one server shouldn't be sent another
-server's cookies.[^tls][^dns][^bgp][^oof] But if an attacker is
+server's cookies.[^tls] But if an attacker is
 clever, they might be able to get *the server* or *the browser* to
 help them steal cookie values.
 
 [^tls]: Well... Our connection isn't encrypted, so an attacker could
     read it from an open Wifi connection. But another *server*
-    couldn't.
-
-[^dns]: Well... Another server could hijack our DNS and redirect our
+    couldn't. Or, another server could hijack our DNS and redirect our
     hostname to a different IP address, and then steal our cookies.
     Some ISPs support DNSSEC, which prevents this, but not all.
-
-[^bgp]: Well... A state-level attacker could announce fradulent BGP
+    Or, a state-level attacker could announce fradulent BGP
     routes, which would send even a correctly-retrieved IP address to
-    the wrong physical computer.
-
-[^oof]: Security is very hard.
+    the wrong physical computer. (Security is very hard.)
```

Comment:
I'm sure this sequence of four footnotes violates Oxford's house style but I do think refactoring it all into one footnote loses a bit of the punch and starts to instead look like an irrelevant tangent.